packages/core/src/components/ui/PickupPoints/PickupPointCard.tsx

```diff
@@ -0,0 +1,37 @@
+import { Icon as UIIcon } from '@faststore/ui'
+import type { PickupPoint } from './PickupPointCards'
+
+export type PickupPointCardProps = {
+  store: PickupPoint
+}
+
+function formatDistance(distance: number) {
+  return `${distance.toFixed(1).replace('.', ',')}km`
+}
+
+export function PickupPointCard({ store }: PickupPointCardProps) {
+  return (
+    <div data-fs-pickup-point-card>
+      <header data-fs-pickup-point-card-header>
+        <UIIcon name="Storefront" />
+        <h3 data-fs-pickup-point-card-header-title>{store.name}</h3>
+        <span data-fs-pickup-point-card-postal-code>
```

Comment:
question: have you removed the 'header' on purpose? If not, remember to remove it also in the stylesheet PickupPointCard/styles.scss
`data-fs-pickup-point-card-**header**-postal-code`

Reply:
😡 my bad, I didn't' double check the find/replace 🙇 fixing it!